### ISSUE TITLE
Mythic Metals Decorations causing crash on startup in 1.20.4

### ISSUE THREAD

**Issue by Gamemassa** [Noaaan/MythicMetalsDecorations#21]
Not sure exactly why the issue is happening, but here is the crash report:

https://gist.github.com/Gamemassa/1546a178124f4d8ea9c6dce1d6c04982

And here is the latest.log:

https://gist.github.com/Gamemassa/db93c2e8d5c8793d88a86c0edd5f0128

**Comment by Noaaan** [comment]
Should be fixed as of 0.6.1

**Comment by Gamemassa** [comment]
> Should be fixed as of 0.6.1

I'm guessing the Curseforge/Modrinth mods are still reviewing it, but I'll test it when it pops up. Also, thanks for the quick reply!

**Comment by Noaaan** [comment]
I have started also releasing Decorations here on GitHub. You can see them in the Releases tab: https://github.com/Noaaan/MythicMetalsDecorations/releases

**Comment by Gamemassa** [comment]
> I have started also releasing Decorations here on GitHub. You can see them in the Releases tab: https://github.com/Noaaan/MythicMetalsDecorations/releases

Oh, ok. That's super convenient. Also, confirmed, definitely working now. Thanks for the quick fix. Out of curiosity, what was the issue? Was it a conflict or something?

**Comment by Noaaan** [comment]
The fix I implemented was related to a Mixin into my own class, which only applies when Lithium is present. 
It implements getters for the size and the contents of the chest inventory using Lithiums interface (`LithiumInventory`).
For some reason one of the fields was not present when running the game normally (this worked fine in my dev environment), so I changed the logic to grab them directly from the chest itself using some small hacks.
